src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -2268,6 +2270,30 @@ exprt c_typecheck_baset::do_special_functions(
     typecheck_function_call_arguments(expr);
     return nil_exprt();
   }
+  else if(identifier == CPROVER_PREFIX "is_freeable")
+  {
+    if(expr.arguments().size() != 1)
+    {
+      throw invalid_source_file_exceptiont{
+        CPROVER_PREFIX "is_freeable expects one operand; " +
+          std::to_string(expr.arguments().size()) + " provided",
+        expr.source_location()};
+    }
+    typecheck_function_call_arguments(expr);
+    return nil_exprt();
+  }
+  else if(identifier == CPROVER_PREFIX "is_freed")
+  {
+    if(expr.arguments().size() != 1)
+    {
+      throw invalid_source_file_exceptiont{
+        CPROVER_PREFIX "is_freed expects one operand; " +
+          std::to_string(expr.arguments().size()) + " provided",
+        expr.source_location()};
+    }
+    typecheck_function_call_arguments(expr);
+    return nil_exprt();
```

Comment:
... and this?

Reply:
same as above. I am just following the pattern set by someone else with `is_fresh`